src/server/index.jsx

```diff
@@ -217,7 +217,7 @@ server
       const bbcOrigin = headers['bbc-origin'];
 
       // Temp log to test upstream change
-      logger.info(`Country code: ${headers['bbc-country'] || 'unknown!'}`);
+      logger.info(`Headers: ${JSON.stringify(headers, null, 2)}`);
```

Comment:
What is the `null, 2` values doing in this scenario?

Reply:
It prettifies the JSON. I thought it was important for viewing the individual headers.